tools/msh2vtu.xml

```diff
@@ -0,0 +1,103 @@
+<tool id="msh2vtu" name="GMSH_To_VTU_Converter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>
+        Convert GMSH mesh files to VTU format using various options for mesh processing.
+    </description>
+    <requirements>
+        <requirement type="package" version="3.6">python</requirement>
+        <requirement type="package" version="0.5">meshio</requirement>
+        <requirement type="package" version="0.1">numpy</requirement>
+        <requirement type="package" version="0.1">ogs</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input_file' input_file.msh &&
+        msh2vtu input_file.msh 
+            -p '$prefix' 
+            -d '$dim' 
+            #if $delz == "true"
+                -z
+            #end if
+            #if $swapxy == "true"
+                -s
+            #end if
+            #if $reindex == "true"
+                -r
+            #end if
+            #if $keep_ids == "true"
+                -k
+            #end if
+            #if $ascii == "true"
+                -a
+            #end if
+            -l '$log_level'
+    ]]></command>
+    <inputs>
+        <param name="input_file" type="data" format="GMSH" label="Input GMSH File"/>
+        <param name="prefix" type="text" label="Filename Prefix" argument="-p">
+            <validator type="regex">^(?!.*[\u003C\u003E])[a-zA-Z0-9_]+$</validator>
+        </param>
+        <param name="dim" type="select" label="Mesh Dimension" argument="-d">
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+        </param>
+        <param name="delz" type="boolean" value="false" label="Delete Z Coordinate" optional="true" argument="-z"/>
+        <param name="swapxy" type="boolean" value="false" label="Swap XY Coordinates" optional="true" argument="-s"/>
+        <param name="reindex" type="boolean" value="false" label="Reindex" optional="true" argument="-r"/>
+        <param name="keep_ids" type="boolean" value="false" label="Keep IDs" optional="true" argument="-k"/>
+        <param name="ascii" type="boolean" value="false" label="ASCII Output" optional="true" argument="-a"/>
+        <param name="log_level" type="select" value="DEBUG" label="Log Level" argument="-l">
+            <option value="DEBUG">DEBUG</option>
+            <option value="INFO">INFO</option>
+            <option value="WARNING">WARNING</option>
+            <option value="ERROR">ERROR</option>
```

Comment:
In a non-optional select the first option is selected by default. Since ERROR should be the defualt here (I guess), we should add `selected="true"` here.